business_requirement_sale_timesheet/models/project_task.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 Tecnativa Victor M.M. Torres>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    business_requirement_deliverable_id = fields.Many2one(
+        comodel_name='business.requirement.deliverable',
+        related='sale_line_id.business_requirement_deliverable_id',
```

Comment:
@Tardo FYI `sale_line_id` at `project.task` is the reason of dependency with `sale_timesheet`